packages/extension/src/routes/Step2.vue

```diff
@@ -204,4 +218,14 @@ export default {
     font-weight: 300;
   }
 }
+
+.btn-skip {
+  margin-top: 24px;
+  font-size: 11px;
```

Comment:
Regarding typography styling we have them predefined.
You should choose the right typography from here, according to `font-size` and other typography properties: https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/mixins/typography.pcss
You can either add the class to your element or use the mixin.